recipes/lixtools/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "lixtools" %}
+{% set version = "2021.6.9.0" %}
+{% set sha256 = "d12e77bfc770ddb9137345c054a6c02afba29bac72768212615126b94b062c4b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - matplotlib
+    - numpy
+    - openpyxl >=3
+    - pandas
+    - pillow
+    - py4xs
+    - python-barcode
+    - qrcode
+
+test:
+  imports:
+    - lixtools
+  commands:  # [not win]
+    - python -c 'import os, pkg_resources; assert os.path.isfile(pkg_resources.resource_filename("lixtools", "plate_label_template.html"))'  # [not win]
+    - python -c 'import os, pkg_resources; assert os.path.isfile(pkg_resources.resource_filename("lixtools", "template_report.ipynb"))'  # [not win]
+
+about:
+  home: https://github.com/NSLS-II-LIX/lixtools
+  license: BSD-3-Clause
+  license_file: LICENSE
+  summary: Software tools for data collection/processing at NSLS-II LiX
+
+extra:
+  recipe-maintainers:
+    - mrakitin
```

Adding @lyang11973 as a core developer of the package. Lin, do you agree to be a maintainer of this conda-forge feedstock?
```suggestion
    - lyang11973
    - mrakitin
```